packages/core/core/src/services/document-service/index.ts

```diff
@@ -29,11 +31,15 @@ export const createDocumentService = (strapi: Strapi): any => {
       return repositories.get(uid)!;
     }
 
-    const repository = createContentTypeRepository(strapi.contentType(uid));
-    repositories.set(uid, repository);
+    const contentType = strapi.contentType(uid);
```

Comment:
I always forget, whats the difference betwen strapi.contentType(uid) and strapi.getModel(uid) ?

Reply:
getModel can return a compo too. we should def avoi dusing it when we can